### PR TITLE
Fix date created

### PR DESCRIPTION
**Pull request by danzvara**

### CODE REVIEW

File: client/meter/consume.js [hunk 1]
```diff
@@ -50,14 +50,19 @@ async function consumeEnergyFromChain(amount) {
...
-
       let count = 0;
       while (energyBalance.lt(amountBigNumber)) {
         await blockchain.updateNow();
         energyBalance = await blockchain.myEnergyBalance();
-        await sleep(1000);
+
+        const buyAmount = (amountBigNumber.minus(energyBalance)).mul(PREBUY_COEF);
+        try {
+          await buy.autoBuy(buyAmount);
```
Comment (fangyi-zhou) [inline client/meter/consume.js]: put sleep here

**Comment by fangyi-zhou** [comment]
Why are you not merging to master?

**Comment by danzvara** [comment]
I branched from your fix-oo7 instead of master

**Comment by danzvara** [comment]
Rebased

**Pull request merged**